File: src/UE4_Packages/FClothingSectionData.java
/**
 * 
 */
package UE4_Packages;

/**
 * @author FunGames
 *
 */
public class FClothingSectionData {
	
	private FGUID assetGUID;
	private int assetLodIndex;

	public FGUID getAssetGUID() {
		return assetGUID;
	}

	public int getAssetLodIndex() {
		return assetLodIndex;
	}

	public FClothingSectionData(FArchive Ar) throws ReadException {
		assetGUID = new FGUID(Ar);
		assetLodIndex = Ar.readInt32();
	}
}

File: src/main/kotlin/me/fungames/jfortniteparse/ue4/assets/objects/meshes/FClothingSectionData.kt
package me.fungames.jfortniteparse.ue4.assets.objects.meshes

import me.fungames.jfortniteparse.ue4.objects.core.misc.FGuid
import me.fungames.jfortniteparse.ue4.reader.FArchive

class FClothingSectionData {
    var assetGuid: FGuid
    var assetLodIndex: Int

    constructor(Ar: FArchive) {
        assetGuid = FGuid(Ar)
        assetLodIndex = Ar.readInt32()
    }

    constructor(assetGuid: FGuid, assetLodIndex: Int) {
        this.assetGuid = assetGuid
        this.assetLodIndex = assetLodIndex
    }
}